spec/arbre/unit/element_spec.rb

```diff
@@ -172,21 +174,22 @@ def helper_method
   describe "rendering to html" do
 
     before  { @separator = $, }
-    after   { $, = @separator }
+    after   { $, = @separator } # rubocop:disable RSpec/InstanceVariable
+
     let(:collection){ element + "hello world" }
 
-    it "should render the children collection" do
-      expect(element.children).to receive(:to_s).and_return("content")
-      expect(element.to_s).to eq("content")
+    it "renders the children collection" do
+      allow(element.children).to receive(:to_s).and_return("children content")
+      expect(element.to_s).to eq("children content")
     end
 
-    it "should render collection when is set the default separator" do
+    it "renders collection when is set the default separator" do
       suppressing_27_warning { $, = "_" }
 
       expect(collection.to_s).to eq("hello world")
     end
```

Comment:
not clear to me what this does

By removing

```rb
    before  { @separator = $, }
    after   { $, = @separator }
```

the spec keeps working

By removing 

```rb
suppressing_27_warning { $, = "_" }
```

The spec keeps working

Also, since now 2.7 is the minimum required version, probably the suppression shouldn't check for the ruby version

Reply:
It's not clear to me either. It was added in #166. If you remove it and run the Ruby 2.7 test build, does it still output a warning?

Reply:
I was able to reproduce the warning, however the fix to remove the offense is probably not worth

```rb
    around do |example|
      old_separator = $,
      example.run
    ensure
      $, = old_separator
    end
```

For the moment I'll disable the offense on that line

Reply:
That's ok. I think it's fine to leave as is. Honestly, I'm not familiar nor understand what that code does. Thanks again for looking into this. I appreciate it.